CMakeLists.txt

```
@@ -5,8 +5,8 @@ project(SlicerOpenLIFU)
 #-----------------------------------------------------------------------------
 # Extension meta-information
 set(EXTENSION_HOMEPAGE "https://github.com/OpenwaterHealth/SlicerOpenLIFU")
-set(EXTENSION_CONTRIBUTORS "Ebrahim Ebrahim (Kitware), , Peter Hollender (Openwater), Sam Horvath (Kitware), Brad Moore (Kitware)")
-set(EXTENSION_DESCRIPTION "An extension for Openwater's Open-LIFU project")
+set(EXTENSION_CONTRIBUTORS "Ebrahim Ebrahim (Kitware), Peter Hollender (Openwater), Sam Horvath (Kitware), Andrew Howe (Kitware), Sadhana Ravikumar (Kitware), Brad Moore (Kitware)")
+set(EXTENSION_DESCRIPTION "A 3D Slicer extension for Openwater's OpenLIFU (Low Intensity Focused Ultrasound) research platform, providing an advanced interface to features found in the OpenLIFU app.")
```

```suggestion
set(EXTENSION_DESCRIPTION "A 3D Slicer extension for Openwater's OpenLIFU (Low Intensity Focused Ultrasound) research platform.")
```

Let's leave it at that, since (a) it's really the python library not the app that we provide an interface for and (b) we do more than just provide a UI -- a lot of workflows are essentially defined here and not in openlifu